meta-flutter/recipes-flutter/engine/flutter-engine_git.bb

```diff
@@ -0,0 +1,10 @@
+LICENSE = "Unknown & MIT & Apache-2.0"
```

Comment:
What is this file?

Reply:
https://github.com/jwinarske/meta-flutter/blob/dunfell/LICENSE

Reply:
I removed this file.